hat/assets/js/apps/Iaso/domains/instances/constants.js

```diff
@@ -94,6 +98,28 @@ export const INSTANCE_MAP_METAS_FIELDS = [
     },
 ];
 
+const OrgUnitLabelHyperLink = ({ value }) => {
+    const currentUser = useCurrentUser();
+    const showOrgUnitLink =
+        userHasOneOfPermissions(
+            [Permission.ORG_UNITS, Permission.ORG_UNITS_READ],
+            currentUser,
+        ) && userHasPermission(Permission.SUBMISSIONS_UPDATE, currentUser);
+    return (
+        <OrgUnitTooltip key={value.id} orgUnit={value} domComponent="span">
+            {showOrgUnitLink ? (
+                <LinkToOrgUnit orgUnit={value} />
+            ) : (
+                <OrgUnitLabel orgUnit={value} withType withSource={false} />
+            )}
+        </OrgUnitTooltip>
+    );
+};
```

Comment:
check on permissions and display of the label is already managed in the `LinkToOrgUnit`